OutOfSchool/OutOfSchool.Migrations/Data/Migrations/OutOfSchoolMigrations/20250409143340_AddStudySubjectRelations.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace OutOfSchool.Migrations.Data.Migrations.OutOfSchoolMigrations
+{
+    /// <inheritdoc />
+    public partial class AddStudySubjectRelations : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: "ProviderId",
+                table: "StudySubjects",
+                type: "UUID(16)",
+                nullable: false,
+                defaultValue: new Guid("00000000-0000-0000-0000-000000000000"));
```

Comment:
it is a `NOT NULL` field that should never have a default property. I have not seen any migration before creating a default value for a foreign key. Yes it would fail a foreign key constraint most of the time, but there might be an error and some one adds a provider with empty Guid and this default value will bind to that provider